kolibri/plugins/user_profile/assets/src/views/ChangeFacility/index.vue

```diff
@@ -93,13 +61,27 @@
         state: computed(() => this.state.context),
       };
     },
+
     computed: {
       ...mapGetters(['session', 'getUserKind']),
+      wrapperStyles() {
+        return {
+          width: '100%',
+          display: 'inline-block',
+          backgroundColor: this.$themePalette.grey.v_100,
+          paddingLeft: '32px',
+          paddingRight: '32px',
+          paddingBottom: '72px',
+          paddingTop: this.appBarHeight + 16 + 'px',
+        };
+      },
     },
+
     created() {
       this.service.start();
       this.service.onTransition(state => {
         const stateID = Object.keys(state.meta)[0];
+        console.log(state);
```

Comment:
Not such a big deal to use console logging as the whole feature will be a work in progress for some time but am surprised that lint checks didn't discover it. Maybe better to remove it?

Reply:
happy to remove it. I had left a couple of debugging lines in the code to facilitate the work of whoever was going to continue with the following screens, but it's clear it's not needed ;)